src/components/BulkEnrollmentPage/table/CourseSearchResultsCells.jsx

```diff
@@ -1,18 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Popover, Button, OverlayTrigger } from '@edx/paragon';
 
 import moment from 'moment';
 
 import { configuration } from '../../../config';
 
+const HREF_TITLE = 'Learn more about this course';
+
 export const CourseNameCell = ({ value, row, enterpriseSlug }) => (
-  <a
-    href={`${configuration.ENTERPRISE_LEARNER_PORTAL_URL}/${enterpriseSlug}/course/${row?.original?.key}`}
-    target="_blank"
-    rel="noopener noreferrer"
+  <OverlayTrigger
+    trigger="click"
+    rootClose
+    key="top"
+    placement="top"
+    overlay={(
+      <Popover id="popover-positioned-top">
+        <Popover.Title>{value}</Popover.Title>
+        <Popover.Content>
+          <div
+            className="desc"
+            /* eslint-disable-next-line */
+            dangerouslySetInnerHTML={{ __html: row?.original?.short_description }}
```

Comment:
i found some prior art in other repos where we do this for course descriptions - so i assume its okay but we can double check.

Reply:
A response I got from Adam S on this when I asked earlier-
> this is how we render the full description (and other HTML-based fields from Algolia in the course about pages in learner portal... i’d say we can trust the data since we’re populating it in the Algolia index ourselves.

Reply:
I made sure other people were using it, because the name certainly is daunting...